### PR TITLE
ENH: Add PR template

### PR DESCRIPTION
**Pull request by richford**
## Proposed changes

This PR adds a PR template that will serve as a starting point for all future PRs. It is intended to be a general purpose PR template for use in **ALL** ROAR repositories. If it is approved and merged for this repository, I will then add it to all other ROAR repos.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
- [x] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
- [x] Linting checks pass with my changes.
- [x] Any existing unit tests pass with my changes.
- [x] Any existing end-to-end tests pass with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
- [x] I have added JSDoc comments as appropriate.
- [x] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).

## Justification of missing checklist items

This PR does not warrant adding any tests.

### CODE REVIEW

**Comment by Zio-4** [comment]
I think we should add a section for images if applicable.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Types of changes
+
+What types of changes does your code introduce?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
```
Comment (lucasxsong) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Perhaps we should add an `other` item for additions like version bumps.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Proposed changes
+
+Describe your changes here. Why are they necessary? If it fixes a bug or
```
Comment (lucasxsong) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: This is more of a stylistic request, but what do you think of changing this block (as well as the one in `Justification of...` and `Further Comments` to be italicized to more easily discern that they are prompts that are meant to be deleted.

**Review by lucasxsong — APPROVED** [review]
General approval, and happy to have this. Think the template will give PR reviewing a lot of useful context and provide good documentation on new additions as well. Just had two small comments. Thanks for writing this up!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Emily-ejag) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Refactoring (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Test improvement (adding or improving tests)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] I have read the [guidelines for contributing](https://github.com/yeatmanlab/roar-dashboard/blob/main/.github/CONTRIBUTING.md).
+- [ ] The changes in this PR are as small as they can be. They represent one and only one fix or enhancement.
+- [ ] Linting checks pass with my changes.
+- [ ] Any existing unit tests pass with my changes.
+- [ ] Any existing end-to-end tests pass with my changes.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] If this PR fixes an existing issue, I have added a unit or end-to-end test that will detect if this issue reoccurs.
+- [ ] I have added JSDoc comments as appropriate.
+- [ ] I have added the necessary documentation to the [roar-docs repository](https://github.com/yeatmanlab/roar-docs).
+
```
Comment (Emily-ejag) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: - [ ] I have shared this PR on channel the roar-pr-reviews
 - [ ] I have linked relevant issues(if any)

Comment (Emily-ejag) [inline .github/PULL_REQUEST_TEMPLATE.md:29]:
```suggestion
 - [ ] I have shared this PR on channel the roar-pr-reviews
 - [ ] I have linked relevant issues(if any)
```

**Review by Zio-4 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (richford) [reply .github/PULL_REQUEST_TEMPLATE.md]: Resolved in latest commit by adding more categories and an "other" category.

Comment (richford) [reply .github/PULL_REQUEST_TEMPLATE.md]: Resolved in latest commit

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (richford) [reply .github/PULL_REQUEST_TEMPLATE.md:49]: Resolved in latest commit

**Comment by richford** [comment]
> I think we should add a section for images if applicable.

Resolved in latest commit by adding a suggestion for images in the HTML comment for the first section.

**Pull request merged**